prov/efa/src/rxr/rxr_env.c

```diff
@@ -136,16 +134,12 @@ void rxr_env_param_get(void)
 			 &rxr_env.shm_cq_read_size);
 	fi_param_get_size_t(&rxr_prov, "inter_max_medium_message_size",
 			    &rxr_env.efa_max_medium_msg_size);
-	fi_param_get_size_t(&rxr_prov, "inter_min_read_message_size",
-			    &rxr_env.efa_min_read_msg_size);
 	fi_param_get_size_t(&rxr_prov, "inter_min_read_write_size",
```

Comment:
we need to do the same for `read_write`.

Reply:
done